pmm-app/src/shared/components/Elements/Table/Table.styles.ts

```diff
@@ -66,5 +66,14 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => {
     checkboxColumn: css`
       width: 20px;
     `,
+    tableClassicExplain: css`
+    table {
+      td {
+        white-space: normal;
+        max-inline-size: 550px;
```

Comment:
I don't think it is good idea to have the same max size. Each column should have it's own max size, and in %.

Reply:
I propose to use 'min-inline-size' instead to avoid linebraking with the 'NULL' and to use maximum possible size for the widthest columns.